presto-product-tests/conf/docker/acid/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '2'
+
+services:
+  hadoop-master:
+    image: 'prestodev/hdp3.1-hive'
+    hostname: hadoop-master
```

Comment:
This extends the `hadoop-master` service definition from `standard.yaml`, so you should not need `hostname`.